include/tvm/runtime/container.h

```diff
@@ -1372,6 +1306,31 @@ class String : public ObjectRef {
    */
   static int memncmp(const char* lhs, const char* rhs, size_t lhs_count, size_t rhs_count);
 
+  /*!
+   * \brief Concatenate two char sequences
+   *
+   * \param lhs Pointers to the lhs char array
+   * \param lhs_size The size of the lhs char array
+   * \param rhs Pointers to the rhs char array
+   * \param rhs_size The size of the rhs char array
+   *
+   * \return The concatenated char sequence
+   */
+  static char* Concat(const char* lhs, size_t lhs_size, const char* rhs, size_t rhs_size) {
+    char* concat = new char[lhs_size + rhs_size + 1];
```

Comment:
This does not resolve the problem of memory leak(since the new char did not get deleted. What I actually meant is to make Concat directly return a String instead, and inside here we use std::string for the space backup.

Reply:
Sorry. I misunderstood.